client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -380,6 +380,8 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 				destination: destString,
 				plan,
 			} );
+
+			setShowUpgradeModal( false );
```

Comment:
This is because the existing modal needs to be closed because the checkout modal is now used. 📝